Library/Homebrew/dev-cmd/unbottled.rb

```diff
@@ -243,4 +256,53 @@ def output_unbottled(formulae, deps_hash, noun, hash, any_named_args)
 
     puts "No unbottled dependencies found!"
   end
+
+  def output_lost_bottles
+    # $ git log --patch -G\^\ \+sha256.\*\ sonoma: --since=@\{\'1\ week\ ago\'\}
+    git_log = %w[git log --patch]
+    git_log << "-G^ +sha256.* #{@bottle_tag}:"
+    git_log << "--since=@{'1 week ago'}"
+
+    bottle_tag_sha_regex = /^[+-] +sha256.* #{@bottle_tag}: /
```

Comment:
Might be good to not have a copy-paste of common part of the regex with the earlier regex.

Reply:
Not sure what you mean by this. Are you suggesting adding another variable to hold shared parts of both regexes?

Reply:
Yeah

Reply:
Sounds good, handled in https://github.com/Homebrew/brew/pull/16115/commits/e48c155f42adeb98b2c6d793e3b2e125d7c2effe